### ISSUE TITLE
flatpak instructions failure

### ISSUE THREAD

**Issue by anarcat** [gnome-keysign/gnome-keysign#75]
Hi,

I am trying to install gnome-keysign from flatpak because the Debian package is currently out of date. Unfortunately, I cannot figure out how that works. Here's me trying to install the flatpakref:

```
anarcat@curie:Downloads(master)$ flatpak install --from https://muelli.cryptobitch.de/tmp/2017-01-29-keysign.flatpakref
The application org.gnome.Keysign depends on runtimes from:
  https://sdk.gnome.org/repo/
Configure this as new remote 'gnome' [y/n]: y
error: Flatpak system operation ConfigureRemote not allowed for user
[1]anarcat@curie:Downloads(master)$ sudo flatpak install --from https://muelli.cryptobitch.de/tmp/2017-01-29-keysign.flatpakref
The application org.gnome.Keysign depends on runtimes from:
  https://sdk.gnome.org/repo/
Configure this as new remote 'gnome' [y/n]: y
error: Unable to load summary from remote org.gnome.Keysign-origin: GPG signatures found, but none are in trusted keyring
```

I would normally expect packages to be installed with an incantation like:

```
flatpak install org.gnome.Recipes
```

... why is this any different?

Thanks!

**Comment by muelli** [comment]
oh, these days it's best to get the flatpak from flathub, i.e. https://flathub.org/apps/details/org.gnome.Keysign.  So the usual instructions, i.e. `flatpak install flathub org.gnome.Keysign` apply.

I rolled that flatpak before flathub (or the app on there) existed, so it's grossly out of date.
Where did you read those instructions?

Does the above work for you?

**Comment by anarcat** [comment]
those instructions are in [the flatpak directory](https://github.com/gnome-keysign/gnome-keysign/tree/master/flatpak) which is explicitely mentioned in the README file.

`flatpak install flathub org.gnome.Keysign` works, of course, thanks for the reference...  it seems the documentation could use an update. :) maybe just remove the pointer to the flatpak directory in the README and point to the above instructions instead?

**Comment by muelli** [comment]
Thanks! Should be fixed with ca62d89
